### DISCUSSION TITLE
Point to specific folder?

### DISCUSSION THREAD

**Discussion by ghost** [abelcheung/rifiuti2#44] [General]
I don't often examine live machines.  Can I extract $I files from an image and then use this application to process just those files?

**Comment by abelcheung** [comment]
Yes. Just extract the whole folder, or even the files individually, and `rifiuti-vista` can examine them. There's no requirement to be on a live system.

**Comment by ghost** [comment]
Okay, I will attempt that.  I didn't see anything in the command line to point to a specific directory so I assumed it just went out and looked for things.  I only want it to look at the files I export from the image.  I will go back and review the command line language.  Thank you.

**Comment by ghost** [comment]
I got it to work.  Thanks for your time..........

**Comment by ghost** [comment]
I'm finding that the file size this is pulling is completely different than what Encase 7 is pulling.  I have a third tool I'm going to use to see what that pulls for file size.

**Comment by ghost** [comment]
Disregard, everything is matching up............

**Comment by abelcheung** [comment]
So everything's working for you now?

I have been thinking about adding option to inspect live system ( see issue #3 ), but so far nothing has materialized yet. There are still too many basic features to fix, and no proper documentation exist as well, so those features would be shelved for later.

**Comment by ghost** [comment]
Everything is good.  Thanks for checking.  
Date: Thu, 7 May 2015 13:08:41 -0700

**Comment by abelcheung** [comment]
OK, feel free to open another report if there is any problem. Thanks for using.
